src/domain/home/hero/hero.module.scss

```diff
@@ -5,29 +5,44 @@
 
 .heroWrapper {
     background-color: $hel-summer;
-    min-height: 30vh;
     display: grid;
     position: relative;
     overflow: visible;
-    background-image: url('../../../assets/icons/svg/kids.png');
-    background-repeat: no-repeat;
-    background-position: center bottom -2px;
-    background-size: 100% auto;
+}
+
+.kidsImageContainer {
+    grid-template-columns: 1fr minmax(auto, $containerMaxWidth) 1fr;
     display: grid;
-    padding-bottom: $largeMargin;
     @include respond-above(lg) {
-        min-height: 35vh;
+        background-image: url('../../../assets/icons/svg/Culture_kids_background@2x.png');
+        background-repeat: repeat-x;
+        background-position: center bottom -2px;
+        background-size: 100% 321px;
     }
 }
 
+.kidsImage {
+    grid-column: 2;
+    @include respond-above(md) {
```

Comment:
These media-queries don't make much sense. (min-width 992px), then next one (max-width 1199px). Instead of using max-width media queries, just make that the default style, then from a certain min-width override what's necessary. Max-width media queries are very rarely needed.